src/software/geom/util.h

```diff
@@ -134,62 +134,17 @@ Segment getSide(const LegacyPolygon<N> &poly, unsigned int i);
 bool collinear(const Point &a, const Point &b, const Point &c);
 
 /**
- * Performs an angle sweep.
- * Suppose in this world, all objects are circles of fixed radius.
- * You are at point \p src, and you want to shoot a ray between \p p1 and \p p2.
- * This function calculates the largest open angle interval that you can shoot.
- *
- * @pre \p p1 must be to the right of \p p2.
- * In other words, if there is a counterclockwise ordering, \p p1 is before \p
- * p2 from \p src's point of view.
- *
- * @pre The angle \p p1, \p src, \p p2 must not be greater than 180 degrees.
- *
- * @pre \p src must not be between \p p1 and \p p2.
- *
- * @param src the location where you are standing.
- *
- * @param p1 the location of the right-hand edge of the target area.
- *
- * @param p2 the location of the left-hand edge of the target area.
- *
- * @param obstacles the coordinates of the centres of the obstacles.
- *
- * @param radius the radii of the obstacles.
- *
- * @return A feasible shot or std::nullopt if there is no feasible shot
- */
-std::optional<Shot> angleSweepCircles(const Point &src, const Point &p1, const Point &p2,
-                                      const std::vector<Point> &obstacles,
-                                      const double &radius);
-
-/**
- * Performs an angle sweep.
- * Suppose in this world, all objects are circles of fixed radius.
- * You are at point \p src, and you want to shoot a ray from a point to an area
- * This function calculates the all open angle intervals that you can shoot.
- *
- * @pre The point \p src can't be within the radius of the obstacle
- *
- * @param src the location where the sweep is centered (think center of a clock)
- *
- * @param p1 the location of the right-hand edge of the target area.
+ * Checks if 3 points are collinear.
```

Comment:
``` suggestion
 * Checks if 3 segments are collinear.
```

Reply:
Good catch.

Changed to 'Checks if 2 Segments are collinear'